### DISCUSSION TITLE
Some needed improvements

### DISCUSSION THREAD

**Discussion by markcmiller86** [community/community#4147] [Discussions]
Looks like I should have posted [this](https://github.community/t/three-improvements-to-discussions/182311) here.

**Comment by michellemerrill** [comment]
> Looks like I should have posted [this](https://github.community/t/three-improvements-to-discussions/182311) here.

@markcmiller86 re: your 3rd bullet point you raised here, the category description does show up when starting a new thread, in the dropdown selector, like here:
<img width="900" alt="Screen Shot 2021-06-10 at 2 03 42 PM" src="https://user-images.githubusercontent.com/73917114/121596915-ed937800-c9f4-11eb-8854-0057cf39b174.png">

**Reply by markcmiller86** [reply]
I guess I was expecting to see it elsewhere but I think that is sufficient. Thanks!
